benchmarks/transport_montecarlo_single_packet_loop.py

```diff
@@ -0,0 +1,24 @@
+"""
+Basic TARDIS Benchmark.
+"""
+
+from benchmarks.benchmark_base import BenchmarkBase
+from tardis.transport.montecarlo import single_packet_loop
+
+
+class BenchmarkMontecarloMontecarloNumbaVpacket(BenchmarkBase):
+    """
+    Class to benchmark the single packet loop function.
+    """
+
+    def time_single_packet_loop(self):
+        single_packet_loop.single_packet_loop(
+            self.packet,
+            self.verysimple_numba_radial_1d_geometry,
+            self.verysimple_numba_model,
```

Comment:
Numba model should be replaced with time explosion if we look here, https://github.com/officialasishkumar/tardis/blob/9a91669fb3a4ceb1a8feaec4bacf1022c7d35253/tardis/transport/montecarlo/single_packet_loop.py#L39

Reply:
This numba model needs to be changed at multiple occurences.

Reply:
done